Header header
int16 rangeInCm
